tests/eclient/testdata/dev_local_info.txt

```diff
@@ -0,0 +1,241 @@
+# Test dev local info
+
+{{define "mngr_port"}}8027{{end}}
+{{define "app_port"}}8028{{end}}
+{{define "token"}}server_token_123{{end}}
+{{define "app_info_status_file"}}/mnt/app-info-status.json{{end}}
+{{define "dev_info_status_file"}}/mnt/dev-info-status.json{{end}}
+{{define "dev_cmd_file"}}/mnt/dev-command.json{{end}}
+{{define "network"}}n1{{end}}
+{{define "ssh"}}ssh -q -o ConnectTimeout=10 -o StrictHostKeyChecking=no -o PasswordAuthentication=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa root@{{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+[!exec:jq] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with a 2 reboot limit
+! test eden.reboot.test -test.v -timewait 100m -reboot=0 -count=2 &
+
+message 'Resetting of EVE'
+eden eve reset
+exec sleep 30
+
+# Create n1 network
+eden -t 1m network create 10.11.12.0/24 -n {{template "network"}}
+test eden.network.test -test.v -timewait 10m ACTIVATED {{template "network"}}
+
+# Deploy local-manager
+eden pod deploy -n local-manager --memory=512MB docker://lfedge/eden-eclient:afaa332 -p {{template "mngr_port"}}:22 --networks={{template "network"}}
```

Comment:
Version of eclient should be updated after image push, and we push images on PR merge right now. So please split PR onto two: one for update of eclient image and the second one for update tests.

Reply:
Ah - that docker tag is what selects the version. Got it.

Reply:
Created a separate https://github.com/lf-edge/eden/pull/762 with eclient.